src/rt/sections_elf_shared.d

```diff
@@ -723,6 +723,11 @@ void scanSegments(in ref dl_phdr_info info, DSO* pdso)
             assert(!pdso._tlsSize); // is unique per DSO
             pdso._tlsMod = info.dlpi_tls_modid;
             pdso._tlsSize = phdr.p_memsz;
+
+            // align to multiple of size_t to avoid misaligned scanning
+            // (size is subtracted from TCB address to get base of TLS)
+            immutable mask = size_t.sizeof - 1;
+            pdso._tlsSize = (pdso._tlsSize + mask) & ~mask;
```

Comment:
But that's just the size, not the beginning of the array. See https://github.com/rainers/druntime/blob/a000c499f477439e1cacbd12b30d7eadd1fea19d/src/rt/sections_elf_shared.d#L969
Any idea how this happens? First few bytes are bools?